systemtests/src/main/java/io/enmasse/systemtest/platform/Kubernetes.java

```diff
@@ -532,6 +534,10 @@ public Pod getPod(String name) {
         return client.persistentVolumeClaims().inNamespace(infraNamespace).withLabels(labels).list().getItems();
     }
 
+    public List<Route> listRoutes(String namespace, Map<String, String> labels) {
```

Comment:
This should be in OpenShift.java I think and the default implementation should be throwing an exception maybe? (CC @kornys )

Reply:
@lulf yup I agree

Reply:
Fixed.